eox_tenant/microsite_aware_functions/enrollments.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
can we rename this file to tenant_aware_functions?

Reply:
Renamed. @felipemontoya